IQKeyboardManager/IQTextView/IQTextView.m

```diff
@@ -98,8 +98,8 @@ -(void)layoutSubviews
 {
     [super layoutSubviews];
 
-    [placeHolderLabel sizeToFit];
     placeHolderLabel.frame = CGRectMake(4, 8, CGRectGetWidth(self.frame)-16, CGRectGetHeight(placeHolderLabel.frame));
+    [placeHolderLabel sizeToFit];
```

Comment:
It's a great thing you find the issue but this isn't the best way to fix the issue I think. Previously I was asking the label to arrange itself to correct size and then getting it's width and height.

But now I think we should ask it's expected size by sizeThatFits method and then write our logic to set it's correct frame.
But now I think we should ask it's expected size by sizeThatFits method and then write our logic to set it's correct frame.